extensions-contrib/opentsdb-emitter/src/main/java/io/druid/emitter/opentsdb/OpentsdbSender.java

```diff
@@ -40,28 +43,40 @@
    */
   private static final String PATH = "/api/put";
   private static final Logger log = new Logger(OpentsdbSender.class);
+  private static final long FLUSH_TIMEOUT = 60000; // default flush wait 1 min
```

Comment:
does this means the emitter will block 1 min waiting for the flush to finish?

Reply:
Yes it is. This is learned from other emitter extensions, e.g graphite-emitter.